synapse/models/transport.py

```diff
@@ -198,34 +423,37 @@ def getModelDefs(self):
                 )),
                 ('transport:air:occupant', {}, (
                     ('type', ('str', {'lower': True}), {
-                        'doc': 'The type of occupant such as pilot, crew or passenger.'}),
+                        'deprecated': True,
+                        'doc': 'Deprecated. Please use transport:occupant.'}),
                     ('flight', ('transport:air:flight', {}), {
-                        'doc': 'The flight that the occupant was aboard.'}),
+                        'deprecated': True,
+                        'doc': 'Deprecated. Please use transport:occupant.'}),
                     ('seat', ('str', {'lower': True}), {
-                        'doc': 'The seat assigned to the occupant'}),
+                        'deprecated': True,
+                        'doc': 'Deprecated. Please use transport:occupant.'}),
                     ('contact', ('ps:contact', {}), {
-                        'doc': 'The contact information of the occupant.'}),
+                        'deprecated': True,
+                        'doc': 'Deprecated. Please use transport:occupant.'}),
```

Comment:
Do we need to deprecate all of the props if the form is deprecated?

Reply:
Dunno? I just did it so any doc strings clarify things on their own.